buffer/src/main/java/io/netty/buffer/api/internal/Statics.java

```diff
@@ -249,4 +262,72 @@ static void writeCharSequence(CharSequence source, Buffer destination, Charset c
         byte[] bytes = source.toString().getBytes(charset);
         destination.writeBytes(bytes);
     }
+
+    static boolean equals(Buffer bufferA, Buffer bufferB) {
```

Comment:
@chrisvest below methods are ported from `ByteBufUtil`

Reply:
In `BufferTestSupport` there's an `assertReadableEquals` method that is no longer necessary now that we have this. Also, even though this code is ported, it would be good to have some tests — in particular, I'd be interested in seeing the equality verified as commutative across different `Buffer` implementations.

Reply:
Done, PTAL